app/script/conversation/ConversationRepository.js

```diff
@@ -1817,7 +1817,7 @@ z.conversation.ConversationRepository = class ConversationRepository {
           protoAsset.set(z.cryptography.PROTO_MESSAGE_TYPE.ASSET_PREVIEW, assetPreview);
           protoAsset.set(
             z.cryptography.PROTO_MESSAGE_TYPE.EXPECTS_READ_CONFIRMATION,
-            this.shouldSendReadReceipt(conversationEntity)
+            this.expectReadReceipt(conversationEntity)
```

Comment:
we could use `option.expectsReadConfirmation` here, since we already have computed that value earlier.